service/src/main/java/bio/terra/workspace/service/workspace/flight/create/azure/ValidateMRGStep.java

```diff
@@ -26,10 +29,33 @@ public ValidateMRGStep(CrlService crlService, AzureConfiguration azureConfig) {
 
   @Override
   public StepResult doStep(FlightContext flightContext) throws InterruptedException {
-    AzureCloudContext azureCloudContext =
-        flightContext
-            .getInputParameters()
-            .get(JobMapKeys.REQUEST.getKeyName(), AzureCloudContext.class);
+    FlightBeanBag appContext = FlightBeanBag.getFromObject(flightContext.getApplicationContext());
+
+    AzureCloudContext azureCloudContext;
+    if (appContext.getFeatureConfiguration().isBpmEnabled()) {
```

Comment:
Do we need this step at all if BPM is enabled? The original intent was to validate the input parameters were a valid MRG. Can we just assume it's valid if it came from BPM?

Reply:
We may want to retain this to ensure that the WSM app registration has access to the MRG. I'm 51/49 in favor of keeping.

Reply:
That's true, technically WSM has a different service principal than BPM. Fine with me to keep, doesn't harm anything.